custom-recipes/wordcloud-recipe/recipe.json

```diff
@@ -0,0 +1,73 @@
+// This file is the descriptor for the Custom code recipe wordcloud-recipe
+{
+    // Meta data for display purposes
+    "meta": {
+        // label: name of the recipe as displayed, should be short
+        "label": "Wordcloud",
+        // description: longer string to help end users understand what this recipe does
+        "description": "Generate wordclouds based on a text column",
+        // icon: must be one of the FontAwesome 3.2.1 icons, complete list here at https://fontawesome.com/v3.2.1/icons/
+        "icon": "icon-cloud"
+    },
+    "kind": "PYTHON",
+    "selectableFromDataset": "input",
+    "inputRoles": [
```

Comment:
We are missing an "INPUT PARAMETERS" separator before text_column
https://github.com/dataiku/dss-plugin-nlp-preparation/blob/f617292ea3a00322317f9a266c8b8e7fd8e2e242/custom-recipes/nlp-preparation-cleaning/recipe.json#L32